indico/modules/rb_new/controllers/backend/common.py

```diff
@@ -43,3 +43,27 @@
     'ne_lat': fields.Float(validate=lambda x: -90 <= x <= 90),
     'ne_lng': fields.Float(validate=lambda x: -180 <= x <= 180)
 }
+
+room_args = {
+    'building': fields.Int(required=True),
+    'capacity': fields.Int(validate=lambda x: 1 <= x, missing=20),
+    'comments': fields.Str(),
+    'division': fields.Str(),
+    'floor': fields.Int(required=True),
+    'full_name': fields.Str(),
```

Comment:
this is a generated property (quite a few others as well), so it cannot be modified directly